nuxeo-core/nuxeo-core-storage-dbs/src/main/java/org/nuxeo/ecm/core/storage/dbs/DBSSession.java

```diff
@@ -845,7 +844,7 @@ protected void remove(String id) {
         // find all sub-docs and whether they're proxies
         Map<String, String> proxyTargets = new HashMap<>();
         Map<String, Object[]> targetProxies = new HashMap<>();
-        Set<String> removedIds = transaction.getSubTree(id, proxyTargets, targetProxies);
+        Set<String> removedIds = transaction.getSubTree(id, proxyTargets, targetProxies, 0);
```

Comment:
What do you think about having:
```
protected Set<String> getSubTree(String id, Map<String, String> proxyTargets, Map<String, Object[]> targetProxies) {
   getSubTree(id, proxyTargets, targetProxies, 0);
}
```

Reply:
We don't need to maintain compatibility, this is a completely internal interface.